backend/config/accounts.go

```diff
@@ -37,6 +37,21 @@ type Account struct {
 	// https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#key-identifiers
 	RootFingerprint []byte                 `json:"rootFingerprint"`
 	Configurations  signing.Configurations `json:"configurations"`
+	// ActiveTokens list the tokens that should be loaded along with the account.  Currently, this
+	// only applies to ETH, and the elements are ERC20 token codes (e.g. "eth-erc20-usdt",
+	// "eth-erc20-bat", etc).
+	ActiveTokens []string `json:"activeTokens"`
+}
+
+// TokenActive returns true if this token is configured to be active.
+// code is the token id, e.g. "eth-erc20-usdt".
+func (account Account) TokenActive(code string) bool {
+	for _, tokenCode := range account.ActiveTokens {
```

Comment:
I know we have only a few tokens at the moment (8?) but in the future, this might be a little performance drain, let's say 100 tokens, also depending on the usage of `TokenActive` which I can't find.

Consider at least sorting the slice when saving so that the search is O(log n) instead of O(n), better yet use a map which is about O(1)? If the idea is for the frontend to call `TokenActive` for every known token, it'll turn into O(n^2). 

I wouldn't bother and suggest optimizing when the time comes but this is a persistent configuration. If you don't do it now, it'll be more work later due to another config migration.

Reply:
Even 100 is instant / not noticeable? 

If you think of many calls to this in a loop could be slow, then we can also sort it before the loop (or we sort it when loading the json) and get log(n) access anyway, without worrying about it when storing it. And what if someone edits the config? Then we also need to sort it at load time anyway to validate it...

But since this function will be deleted anyway (unused), there is nothing to do anyway, right?

Reply:
Yeah, nothing to do if removed. I was commenting preemptively :)